plugins/modules/iam_policy.py

```diff
@@ -35,7 +35,6 @@
   policy_json:
     description:
       - A properly json formatted policy as string.
-      - See U(https://github.com/ansible/ansible/issues/7005#issuecomment-42894813) on how to use it properly.
```

Comment:
Note for the reader, we switched to the "JSON" type prior to Ansible 2.10 which makes this reference about Ansible mangling templated JSON strings moot.